src/index.js

```diff
@@ -32,77 +30,69 @@ api.get = (token, options, getCallback) => {
 	const github = githubFactory(token);
 
 	// Get all repositories
-	github.repos
-		.list({type: 'public'})
-		.then((repos) => {
-			// Keep only forks
-			let forks = repos.filter(({fork}) => fork);
+	try {
+		const repos = await github.paginate(github.repos.listForAuthenticatedUser, {
+			type: 'public'
+		});
+		// Keep only forks
+		let forks = repos.filter(({fork}) => fork);
 
-			// Keep only forks owned by a specified user
-			if (options.user) {
-				forks = forks.filter(({owner}) => owner.login === options.user);
-			}
+		// Keep only forks owned by a specified user
+		if (options.user) {
+			forks = forks.filter(({owner}) => owner.login === options.user);
+		}
 
-			let countDoneForks = 0;
+		let countDoneForks = 0;
+		options.progress({
+			countInspected: 0,
+			totalToInspect: forks.length
+		});
+		const forkDone = (fork) => {
+			countDoneForks += 1;
 			options.progress({
-				countInspected: 0,
+				countInspected: countDoneForks,
+				lastInspected: fork.name,
 				totalToInspect: forks.length
 			});
-			const forkDone = (fork) => {
-				countDoneForks += 1;
-				options.progress({
-					countInspected: countDoneForks,
-					lastInspected: fork.name,
-					totalToInspect: forks.length
-				});
-			};
-
-			// Keep only useless forks
-			async.filter(
-				forks,
-				(fork, filterCallback) => {
-					shouldDeleteFork(github, fork, (error, result) => {
-						if (error) {
-							options.warnings(
-								`Failed to inspect ${fork.name}, skipping`,
-								error
-							);
-							result = false;
-						}
-
-						forkDone(fork);
-						filterCallback(null, result);
-					});
-				},
-				(error, forksToDelete) => {
-					if (error) {
-						return getCallback(error);
-					}
+		};
 
+		// Keep only useless forks
+		const processedForks = forks.map(async (fork) => {
+			try {
+				if (await shouldDeleteFork(github, fork)) {
 					// Map to our simple objects
-					const response = forksToDelete.map((fork) => ({
+					return {
 						owner: fork.owner.login,
 						repo: fork.name,
 						url: fork.html_url
-					}));
-					getCallback(null, response);
+					};
 				}
-			);
-		})
-		.then(null, (error) => {
-			getCallback(error);
+			} catch (error) {
+				options.warnings(`Failed to inspect ${fork.name}, skipping`, error);
+			} finally {
+				forkDone(fork);
+			}
 		});
+
+		const forksToDelete = await Promise.all(processedForks);
+		getCallback(null, forksToDelete.filter(Boolean));
+	} catch (error) {
+		getCallback(error);
+	}
 };
 
-api.remove = (token, repos, removeCallback) => {
+api.remove = async (token, repos, removeCallback) => {
 	const github = githubFactory(token);
-	async.each(
-		repos,
-		(repo, callback) => {
-			github.repos.delete({owner: repo.owner, repo: repo.repo}, callback);
-		},
-		removeCallback
-	);
+	try {
+		await Promise.all(
+			repos.map((repo) =>
+				github.repos.delete({owner: repo.owner, repo: repo.repo})
+			)
+		);
+		removeCallback();
+	} catch (error) {
+		removeCallback(error);
```

Comment:
Same as for api.get above:

```suggestion
		removeCallback();
		return;
	} catch (error) {
		removeCallback(error);
		throw error;
```